pkg/identity/keystone/token_getter.go

```diff
@@ -22,9 +22,12 @@ import (
 	"github.com/gophercloud/gophercloud"
 	"github.com/gophercloud/gophercloud/openstack"
 	tokens3 "github.com/gophercloud/gophercloud/openstack/identity/v3/tokens"
+	"github.com/gophercloud/utils/client"
 	"io/ioutil"
 	certutil "k8s.io/client-go/util/cert"
+	openstack_provider "k8s.io/cloud-provider-openstack/pkg/cloudprovider/providers/openstack"
```

Comment:
Please don't use either under_scores or mixedCaps for package name, I suggest to use `cpo` as other files do.

Reply:
ok, will fix this.